src/App.jsx

```diff
@@ -1,60 +1,178 @@
-import "./App.css"
+import { useState } from "react";
+import "./App.css";
 
 function App() {
```

Comment:
function naming convention in JS / react: functions are camelCase (except COMPONENT function which are PascalCase)